castai/provider.go

```diff
@@ -34,6 +34,7 @@ func Provider() *schema.Provider {
 		DataSourcesMap: map[string]*schema.Resource{
 			"castai_credentials": dataSourceCastaiCredentials(),
 			"castai_cluster":     dataSourceCastaiCluster(),
+			"castai_gcp_user_policies":  dataSourceGcpPolicies(),
```

Comment:
name this `castai_gke_user_policies` - we're building kubernetes provider-specific DS, not cloud-specific DS

Reply:
good point, renamed gcp -> gke